assets/src/application/ui/input/InlineEditInput/InlineEditText.tsx

```diff
@@ -1,10 +1,49 @@
 import React from 'react';
+import Dotdotdot from 'react-dotdotdot';
+
+import { Textfit } from 'react-textfit';
 
 import { TextProps } from './types';
-import { InlineEdit } from '@infraUI/inputs';
+import { InlineEdit, InlineEditPreviewProps } from '@infraUI/inputs';
+
+const Preview: React.FC<InlineEditPreviewProps> = ({ fitText, isEditing, onRequestEdit, value }) => {
+	if (isEditing) {
+		return null;
+	}
+
+	if (value.length > 40) {
```

Comment:
40 chars title:

![image](https://user-images.githubusercontent.com/1477453/81811352-2da3d500-952d-11ea-93ad-a27a4b19460e.png)

Reply:
The titles don't need to be crammed into one line. Form #2783:

> names over 50 characters in length were truncated so that they do not span more than 2 (or 3?) lines of text

I think two lines is good and we don't want the font size to get too small as the title should still be the most prominent datum in the main card body. The above looks like the title font is smaller than the desctiprion font.

Reply:
made 18px(16px is the desc font-size) the minimum font-size and 30 chars the lowest limit before it moves into 2 lines


![Peek 2020-05-13 21-06](https://user-images.githubusercontent.com/1477453/81848408-bdfa0e00-955d-11ea-84f4-1d08040d7e18.gif)

Reply:
2 lines is max:

![image](https://user-images.githubusercontent.com/1477453/81848784-509aad00-955e-11ea-9c33-a9cecd433a83.png)